.requirements

```diff
@@ -2,10 +2,10 @@ KONG_PACKAGE_NAME=kong
 KONG_CONFLICTS=kong-enterprise-edition
 KONG_LICENSE="ASL 2.0"
 
-RESTY_VERSION=1.19.3.2
+RESTY_VERSION=1.19.9.1
 RESTY_LUAROCKS_VERSION=3.7.0
 RESTY_OPENSSL_VERSION=1.1.1k
 RESTY_PCRE_VERSION=8.44
 LIBYAML_VERSION=0.2.5
 KONG_GO_PLUGINSERVER_VERSION=v0.6.1
-KONG_BUILD_TOOLS_VERSION=4.20.0
+KONG_BUILD_TOOLS_VERSION=master
```

Comment:
not pinning to a tag has pros/cons. I'm not saying this needs to change or is wrong just be aware of them when making the change ;)

Reply:
@hutchic this is known, I pinned it temporary to `master` while we did not have a release. Thus I marked this as `do not merge`, now I am going to upgrade the version as the patches were merged and new release was published. I hope this explains why. :-) But thanks for your attention to detail, I like it!